packages/console/src/components/CreateTenantModal/index.tsx

```diff
@@ -66,18 +71,37 @@ function CreateTenantModal({ isOpen, onClose, skipPlanSelection = false }: Props
   const cloudApi = useCloudApi();
 
   const createTenant = async (data: CreateTenantData) => {
-    void trySafe(async () => {
-      const { name, tag } = data;
-      const newTenant = await cloudApi.post('/api/tenants', { body: { name, tag } });
-      onClose(newTenant);
-    });
+    const { name, tag } = data;
+    const newTenant = await cloudApi.post('/api/tenants', { body: { name, tag } });
+    onClose(newTenant);
```

Comment:
Why removing `trySafe`?

Reply:
Remove the trySafe wrapper for the tenant creation request since now the error will be handled by the useCloudApi hook.

Reply:
ok